File: EFCore Security Applications/EFCoreSecurityAndroidDemo/app/src/main/java/com/devexpress/efcoresecurity/efcoresecuritydemo/businessobjects/Department.java
package com.devexpress.efcoresecurity.efcoresecuritydemo.businessobjects;

import java.util.ArrayList;
import java.util.Collection;

/**
 * Created by neroslavskiy.a on 4/6/2016.
 */
public class Department extends BaseSecurityEntity {
    public String Title;
    public String Office;
    public ArrayList<Contact> Contacts;
}

File: EFCoreSecurityDemos/EFCoreSecurityConsoleDemo/EFCoreSecurityConsoleDemo/DataModel/Department.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using DevExpress.EntityFramework.SecurityDataStore.Security.BaseSecurityEntity;

namespace EFCoreSecurityODataService.DataModel {
    public class Department : BaseSecurityEntity {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Office { get; set; }
        public List<Contact> Contacts { get; set; }
        public Department() {
            Contacts = new List<Contact>();
        }
    }
}